k8s/cluster.yaml

```diff
@@ -218,8 +218,8 @@ spec:
     testground.nodetype: infra
   image: kope.io/k8s-1.17-debian-stretch-amd64-hvm-ebs-2020-01-17
   machineType: c5.2xlarge
-  maxSize: 2
-  minSize: 2
+  maxSize: 3
+  minSize: 3
```

Comment:
So we're reserving an entire c5.2xlarge machine for Influx?

Reply:
Maybe we can scale down Redis a bit, and just fit it next to Redis.

Currently we are pretty much reserving an instance for:
- Redis
- Prometheus

Influx doesn't fit on any of these with the current resource requirements we have.

Reply:
Scaled down to 2.